packages/delivery-electron/delivery.js

```diff
@@ -1,8 +1,12 @@
 const { createHash } = require('crypto')
+const { createReadStream } = require('fs')
+const { basename } = require('path')
+const { createGzip } = require('zlib')
 const payload = require('@bugsnag/core/lib/json-payload')
 const PayloadQueue = require('./queue')
 const PayloadDeliveryLoop = require('./payload-loop')
 const NetworkStatus = require('./network-status')
+const FormData = require('form-data')
```

Comment:
I guess this is available because Electron uses it internally?

Should we add this as a dependency of this module so we're not necessarily reliant on the version Electron uses? e.g. if a major version comes out with a breaking change and Electron updates to use it